### ISSUE TITLE
udev rules don't trigger properly

### ISSUE THREAD

**Issue by udeved** [gentoo/eudev#55]
Hi, 

I am on Arch Linux, and I use a separarte /usr partition.

I compile eudev with following configuration:

```
./configure  \
            --prefix=/usr \
            --sysconfdir=/etc \
            --sbindir=/sbin \
            --bindir=/bin \
            --libdir=/usr/lib \
            --enable-libkmod \
            --enable-modules \
            --enable-gudev \
            --enable-introspection \
            --with-modprobe=/sbin/modprobe \
            --with-rootprefix=/usr \
            --with-firmware-path=/usr/lib/firmware/updates:/lib/firmware/updates:/usr/lib/firmware:/lib/firmware
```

I update initrd after eudev install, I boot, openrc takes over, udev-init-scripts start up, but no additional kernel modules are loaded besides the ones loaded by initrd.

It doesn't make a difference if I set "enable-split-usr", the result is the same.

The bin dirs go on root partition so udev-init-scripts can run, since they don't like symlinks.

In summary, I end up with a booting system, mounted partitions, but for some reason, no additional kernel modules are loaded.

Here is a screenshot:

http://openrc4arch.site40.net/pool/archlinux/eudev.png

http://openrc4arch.site40.net/pool/archlinux/eudev1.png

http://openrc4arch.site40.net/pool/archlinux/eudev2.png
